### ISSUE TITLE
Alt/Option + Scroll Wheel for Zoom shortcut

### ISSUE THREAD

**Issue by Gazook89** [naturalcrit/homebrewery#3730]
### Your idea:

Affinity programs (Designer, Photo, Publisher) and Adobe programs (InDesign, Photoshop, etc) use `Alt/Option` + `Scroll Wheel` with the cursor in the window to zoom the window.  I'd like to add that functionality into HB, when the cursor is in the iFrame. 

As I've been doing a lot of work in Publisher to create a new Theme for HB, I keep switching to HB and trying the shortcut for zooming and getting frustrated it doesn't work.  Now we have the zoom functionality in, it *should* be easy to add such a shortcut.  

It should be limited to the iframe (doing the shortcut on the Editor shouldn't change anything).

**Comment by 5e-Cleric** [comment]
We could add to this using right and left arrow to navigate pages, as most if not all pdf viewers do.

**Comment by Gazook89** [comment]
I wouldn't want to do any keyboard navigation in the same PR that covers my request above.  I believe it keyboard navigation is more complicated in design.

With my proposed feature, we can listen for mousescroll events inside the iFrame or Preview pane which is just based on the mouse position.

With keyboard interactions, we would need to be sure that the Preview Pane/iFrame has focus set on it, which I don't believe we really have right now.  Without focus, we don't know if pressing left or right arrows should move the cursor in the Editor or not (or any other place).  Maybe I'm wrong about the difficulty, but my brain just has some flashing lights saying "context sensitive hotkeys are hard when you have no way of focusing the context".

**Comment by dbolack-ab** [comment]
Did ya'll get this to an actionable conclusion?

**Comment by 5e-Cleric** [comment]
Yes, we just need an event listener for mousescroll events, when that happens, check if alt is being pressed and if the mouse is over the preview pane, if it is, update the zoom, so the event listener can happen in brewRenderer.jsx, that should do it.
